src/System.Buffers/src/Configurations.props

```diff
@@ -6,13 +6,13 @@
       netstandard;
       netcoreapp2.0-Windows_NT;
       netcoreapp2.0-Unix;
-      uap-Windows_NT;
```

Comment:
I think this will cause a degradation when the `netstandard` asset is used.  It looks like the `netstandard` implementation is significantly different in that it doesn't typeforward.  I think the package should include some UAP asset to ensure that it doesn't degrade the inbox impl when winning conflict.

Reply:
See https://github.com/dotnet/corefx/pull/23719 and https://github.com/dotnet/corefx/pull/23719/commits/75962dae7f338445b57d7d276d018774e1dd2e46 for descriptions of this situation